pkg/resiliency/retry_test.go

```diff
@@ -0,0 +1,56 @@
+package resiliency
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestResiliencyFunctions(t *testing.T) {
```

Comment:
Please add test cases for gRPC status codes (which are < 100)

Reply:
done.